test/Serilog.Tests/Serilog.Tests.csproj

```diff
@@ -29,7 +29,7 @@
     <DefineConstants>$(DefineConstants);APPDOMAIN;REMOTING;GETCURRENTMETHOD</DefineConstants>
   </PropertyGroup>
   <PropertyGroup Condition=" '$(TargetFramework)' == 'net46' ">
-    <DefineConstants>$(DefineConstants);ASYNCLOCAL</DefineConstants>
+    <DefineConstants>$(DefineConstants);ASYNCLOCAL;GETCURRENTMETHOD</DefineConstants>
```

Comment:
It lights up an additional test that we only run when `GetCurrentMethod()` is available (just noticed the inconsistency).